shared/src/ledger/ibc/storage.rs

```diff
@@ -69,6 +69,7 @@ pub enum IbcPrefix {
     Ack,
     Event,
     Unknown,
+    NonIbc,
```

Comment:
I think it may be a little cleaner to not add this type and instead change the return the of `ibc_prefix` to `Option<IbcPrefix>` and return `None` for non-IBC keys